src/DotLiquid.Tests/RegexpTests.cs

```diff
@@ -18,20 +18,11 @@ public void TestAllRegexesAreCompiled()
             var assembly = typeof (Template).GetTypeInfo().Assembly;
             foreach (Type parent in assembly.GetTypes())
             {
-                foreach (var t in parent.GetTypeInfo().GetFields(BindingFlags.Instance | BindingFlags.Static | BindingFlags.Public | BindingFlags.NonPublic))
+                foreach (var t in parent.GetTypeInfo().GetFields(BindingFlags.Static | BindingFlags.Public | BindingFlags.NonPublic))
                 {
                     if (t.FieldType == typeof(Regex))
                     {
-                        if (t.IsStatic)
-                        {
-                            Assert.AreNotEqual(0, RegexOptions.Compiled & ((Regex) t.GetValue(null)).Options);
-                        }
-                        else
-                        {
-                            Assert.AreNotEqual(0, RegexOptions.Compiled & ((Regex)t.GetValue(parent)).Options);
-                        }

```

Comment:
Instance variables need an instance to be created. parent is a Type not an instance of it, and without knowing the constructor parameters this is not possible. Killed non-working code entirely to allow this test to pass.